src/main.rs

```diff
@@ -419,12 +419,24 @@ fn real_main() -> Result<(), miette::Report> {
         })
         .transpose()?;
 
+    let cli_metacfg = cli.config.as_ref().map(|path| {
+        MetaConfigInstance {
+            version: Some(1),
```

Comment:
I think this can be `None`, in which case it'll fall back to the default config. That way if a different config is specified in `Cargo.toml`, it will be respected.

Reply:
Yes, I think that's already what happens (with the fix below to precedence of --store-path), unless I am misunderstanding your comment.

Reply:
My comment is that right now it's treated as an explicitly specified version, rather than one which falls back, and it might be more elegant for it to fall back instead of being explicit. Doesn't really matter right now though I suppose, as there's only one version.